### ISSUE TITLE
Don't re-upload directory content if only folder name is changed.

### ISSUE THREAD

**Issue by MeijeSibbel** [GooBox/goobox-sync-storj#102]

**Comment by kaloyan-raev** [comment]
This depends on the file rename feature of the Storj bridge.

**Comment by jasonwee** [comment]
Yes, reproducible, see https://gist.github.com/jasonwee/1369494623c3629bdc2a6ef903a5cc14, it looks like first, delete directory and then delete the files in cloud. Follow by create cloud directory and its file for upload.

Looks like there is no `rename` method in this project too https://github.com/storj/java-libstorj

**Comment by MeijeSibbel** [comment]
See the original Libstorj-java repo on the storj github. There is a issue open for it there.

**Comment by MeijeSibbel** [comment]
https://github.com/storj/libstorj/pull/433

**Comment by jasonwee** [comment]
Thanks @MeijeSibbel  yes, so the libstorj has to fix first, then java-libstorj and last goobox-sync-storj.
